tests/test_models/stochastic.py

```diff
@@ -1,21 +1,39 @@
-"""Define example model specifications."""
+"""Example specifications of a simple Phelps-Deaton style stochastic model."""
 
 import jax.numpy as jnp
-
 import lcm
 
-N_CHOICE_GRID_POINTS = 500
-N_STATE_GRID_POINTS = 100
+# ======================================================================================
+# Numerical parameters and constants
+# ======================================================================================
+
+N_GRID_POINTS = {
+    "states": 100,
+    "choices": 200,
+}
 
+# ======================================================================================
+# Model functions
+# ======================================================================================
 
+
+# --------------------------------------------------------------------------------------
+# Utility function
+# --------------------------------------------------------------------------------------
 def utility(consumption, working, health, partner, delta, gamma):  # noqa: ARG001
```

Comment:
Would be neat to know what delta and gamma are supposed to do.

Reply:
Good point. Renamed it to "disutility_of_work".